src/olympia/amo/tests/test_search.py

```diff
@@ -212,60 +212,60 @@ def test_len(self):
 
     def test_gte(self):
         qs = Addon.search().filter(type__in=[1, 2], status__gte=4)
-        filters = qs._build_query()['query']['filtered']['filter']
+        filters = qs._build_query()['query']['bool']['filter']['bool']
         # Filters:
         # {'and': [
         #     {'in': {'type': [1, 2]}},
         #     {'range': {'status': {'gte': 4}}},
         # ]}
-        assert filters.keys() == ['and']
-        assert {'in': {'type': [1, 2]}} in filters['and']
-        assert {'range': {'status': {'gte': 4}}} in filters['and']
+        assert filters.keys() == ['must']
+        assert {'in': {'type': [1, 2]}} in filters['must']
+        assert {'range': {'status': {'gte': 4}}} in filters['must']
 
     def test_lte(self):
         qs = Addon.search().filter(type__in=[1, 2], status__lte=4)
-        filters = qs._build_query()['query']['filtered']['filter']
+        filters = qs._build_query()['query']['bool']['filter']['bool']
         # Filters:
         # {'and': [
         #     {'in': {'type': [1, 2]}},
         #     {'range': {'status': {'lte': 4}}},
         # ]}
-        assert filters.keys() == ['and']
-        assert {'in': {'type': [1, 2]}} in filters['and']
-        assert {'range': {'status': {'lte': 4}}} in filters['and']
+        assert filters.keys() == ['must']
+        assert {'in': {'type': [1, 2]}} in filters['must']
+        assert {'range': {'status': {'lte': 4}}} in filters['must']
 
     def test_gt(self):
         qs = Addon.search().filter(type__in=[1, 2], status__gt=4)
-        filters = qs._build_query()['query']['filtered']['filter']
+        filters = qs._build_query()['query']['bool']['filter']['bool']
         # Filters:
         # {'and': [
         #     {'in': {'type': [1, 2]}},
         #     {'range': {'status': {'gt': 4}}},
         # ]}
-        assert filters.keys() == ['and']
-        assert {'in': {'type': [1, 2]}} in filters['and']
-        assert {'range': {'status': {'gt': 4}}} in filters['and']
+        assert filters.keys() == ['must']
+        assert {'in': {'type': [1, 2]}} in filters['must']
+        assert {'range': {'status': {'gt': 4}}} in filters['must']
 
     def test_lt(self):
         qs = Addon.search().filter(type__in=[1, 2], status__lt=4)
-        filters = qs._build_query()['query']['filtered']['filter']
+        filters = qs._build_query()['query']['bool']['filter']['bool']
         # Filters:
         # {'and': [
         #     {'range': {'status': {'lt': 4}}},
         #     {'in': {'type': [1, 2]}},
         # ]}
```

Comment:
Before someone comments on this… the comments will be updated accordingly. I really just want to check if we have tests that actually verify these filters and functionalities on the ES side of things. Some do but I don't know if we have explicit tests that mirror the `_build_query` tests.